webboot/webboot.go

```diff
@@ -153,7 +148,16 @@ func main() {
 		log.Fatal(err)
 	}
 
-	err = write(read, filename)
+	file, err := os.OpenFile("/dev/pmem0", os.O_APPEND|os.O_WRONLY, 0600)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	body, err := ioutil.ReadAll(read)
```

Comment:
do not read the body here.

Reply:
I realized I do not need to read the body at all. I will get rid of this.